packages/cysync-core-constants/src/i18n/lang/en.json

```diff
@@ -1114,7 +1114,8 @@
         "subtext": "Reconnect the device and try again and if the problem persists, contact Cypherock support for assistance"
       }
     },
-    "default": "Some internal error occurred"
+    "default": "Some internal error occurred",
+    "videoPlaybackError": "No Internet connection. Please check your connection and try again"
```

Comment:
We already have `SER_0001`, consult with akshit to see if we really need a different message for this

Reply:
Akshit told me to use the new message